Userland/Libraries/LibGfx/GradientPainting.cpp

```diff
@@ -213,18 +213,28 @@ static auto create_conic_gradient(ReadonlySpan<ColorStop> color_stops, FloatPoin
     };
 }
 
-static auto create_radial_gradient(IntRect const& physical_rect, ReadonlySpan<ColorStop> color_stops, IntPoint center, IntSize size, Optional<float> repeat_length)
+static auto create_radial_gradient(IntRect const& physical_rect, ReadonlySpan<ColorStop> color_stops, IntPoint center, IntSize size, Optional<float> repeat_length, Optional<float> rotation_angle = {})
 {
     // A conservative guesstimate on how many colors we need to generate:
     auto max_dimension = max(physical_rect.width(), physical_rect.height());
     auto max_visible_gradient = max(max_dimension / 2, min(size.width(), max_dimension));
     GradientLine gradient_line(max_visible_gradient, color_stops, repeat_length);
     auto center_point = FloatPoint { center }.translated(0.5, 0.5);
+    AffineTransform rotation_transform;
+    if (rotation_angle.has_value()) {
+        auto angle_as_radians = rotation_angle.value() * static_cast<float>((M_PI / 180));
```

Comment:
Use AK instead of the C M_PI macro:
```suggestion
        auto angle_as_radians = rotation_angle.value() * (AK::Pi<float>  / 180);
```